src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -195,7 +201,7 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
                     resourceData.satelliteResources ??= new Dictionary<string, ResourceHashesByNameDictionary>(StringComparer.OrdinalIgnoreCase);
 
                     if (!IsTargeting80OrLater())
-                        resourceName = assetTraitValue + "/" + resourceName;
+                        resourceRoute = assetTraitValue + "/" + resourceRoute;
```

Comment:
Is this correct?

Won't this result in something like `en-US/_framework/assembly.wasm` as opposed to `/_framework/en-US/assembly.wasm`

Maybe this is something that needs to be resolved in the relevant `ComputeWasm*` (by setting the appropriate relative path) and avoid having to do path manipulations here.

Reply:
We are not putting `_framework` into the boot config. It contains `es-US/assembly.wasm` and `_framework` added based on location of `dotnet.js`